app/views/admin/customers/index.html.haml

```diff
@@ -35,7 +35,7 @@
 
   .row{ 'ng-if' => 'shop_id && RequestMonitor.loading' }
     .sixteen.columns.alpha#loading
-      %img.spinner{ src: image_path("spinning-circles.svg") }
+      = render partial: "components/loading"
```

Comment:
I'd rather name it `component/spinner`. That's a name for a "thing" rather than a state (loading).

Reply:
Yes, agree, will correct it.

Reply:
corrected via 8399b82